src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -76,6 +76,30 @@ public function get($uri, array $headers = [])
     }
 
     /**
+     * Visit the given URI with a GET request with content type of application/json.
+     *
+     * @param  string  $uri
+     * @param  array  $data
+     * @param  array  $headers
+     * @return $this
+     */
+    public function getJson($uri, array $data = [], array $headers = [])
+    {
+        $content = json_encode($data);
+        $headers['CONTENT_TYPE'] = 'application/json';
+        $headers['CONTENT_LENGTH'] = mb_strlen($content);
+        if (isset($headers['Accept']) === false) {
```

Comment:
@samlev Typesafe comparison (`===`) is used when we can't totally rely on the type of the value that is checked to avoid false positives. However, `isset()` will always return a boolean so we don't need that.